frontend/src/components/SamplesList.tsx

```diff
@@ -27,30 +32,28 @@ interface ISampleListProps {
   columnDefs: ColDef[];
   defaultColDef: ColDef;
   getRowData: (samples: Sample[]) => any[];
-  height: number;
-  setUnsavedChanges?: (val: boolean) => void;
-  searchVariables?: SampleWhere;
-  filter: (searchVal: string) => SampleWhere;
+  setUnsavedChanges?: (unsavedChanges: boolean) => void;
+  parentWhereVariables?: SampleWhere;
+  refetchWhereVariables: (parsedSearchVals: string[]) => SampleWhere;
   exportFileName?: string;
 }
 
-export const SamplesList: FunctionComponent<ISampleListProps> = ({
+export default function SamplesList({
```

Comment:
In React/JS, we can define a function component either with `const` or `function`. There are pros and cons to each approach. I'm changing this to `function` to (1) match how we define function components in other places of the codebase and (2) improve readability for our team members who also work in non-React/JS codebases like Java.